src/util-var.h

```diff
@@ -57,9 +57,21 @@ typedef struct XBit_ {
     uint32_t expire;
 } XBit;
 
+// A list of variables we try to resolve.
+// Helps to detect recursive declarations.
+typedef struct ResolvedVariable_ {
+    char var_name[255];
```

Comment:
super minor, but shouldn't this be 256? Due to alignment the struct will still have the same size now :)